tests/test_arcaflow_plugin_stressng.py

```diff
@@ -61,18 +61,17 @@ def test_functional_cpu(self):
                 print(e)
 
         self.assertEqual(yaml.safe_load(result), reference)
-        workload_params = stressng_schema.WorkloadParams(stress, True)
-        res = stressng_plugin.stressng_run(self.id(), workload_params)
+        res = stressng_plugin.stressng_run(self.id(), stress)
         self.assertIn("success", res)
         self.assertEqual(res[1].cpuinfo.stressor, "cpu")
-        self.assertGreaterEqual(math.ceil(res[1].cpuinfo.wall_clock_time), 10)
+        self.assertGreaterEqual(math.ceil(res[1].cpuinfo.wall_clock_time), 5)
```

Comment:
The (now) `5` is a magic number which should be referenced using a symbolic constant, so that we're assured of a consistent value in all the places that it is used.  (If it had been coded this way originally, changing it from `10` to `5` would have required changing it in only one place....)

Ditto line 93. et al. -- if you make the constant global, you only need to define/set it in one place.